package.json

```diff
@@ -25,16 +25,17 @@
     "extract-error-codes": "lerna run --parallel extract-error-codes",
     "framer:build": "yarn workspace framer build",
     "jsonlint": "node scripts/jsonlint.js",
-    "lint": "eslint . --cache --report-unused-disable-directives --ext .js,.ts,.tsx",
-    "lint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx",
+    "lint": "yarn eslint && yarn jsonlint",
+    "eslint": "eslint . --cache --report-unused-disable-directives --ext .js,.ts,.tsx",
+    "eslint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx",
```

Comment:
Repeating original comment since github dropped commit reviews. Please leave these command name s unless there is a concrete issue. Changing workspace commands disrupts existing workflows.

Reply:
This is a change that was asked by @dtassone, I think that we should still be willing to accept disruption of existing workflow if it translate into improvements. 

The improvement in our case is about:
1. Bundling all the lint commands inside one that can be run in development. In practice I personally never use them, I prefer switching to a different task, waiting for the answer of the CI. 
2. Breaking down each lint command into a descriptive name, and running each lint command in isolation in the CI.

Now, it's a small detail, that I think is more about creating precedence for being flexible and welcoming to changes.

Reply:
Add a new lint:all taks then

Reply:
> welcoming to changes

Nobody welcomes breaking changes.

Reply:
> Breaking down each lint command into a descriptive name, and running each lint command in isolation in the CI.

What is the improvement here?

> Bundling all the lint commands inside one that can be run in development. In practice I personally never use them, I prefer switching to a different task, waiting for the answer of the CI.

You don't use it. I do. Why do we prioritize what you want?

Reply:
Ok, I have reverted